.github/workflows/node-step-docker-build-and-push-image.yml

```diff
@@ -35,6 +35,10 @@ on:
         required: false
         type: string
         default: ${{ github.actor }}
+      ENV_VARIABLES:
```

Comment:
github non permette di passare le env variables tra workflow riutilizzabili (quindi questi che usiamo), quindi ho dovuto aggiungere un input in più, json, che permetta l'aggiunta di N variabili (bruttino ma non ho trovato altre soluzioni carine...).

Retrocompatibile, quindi non causa problemi